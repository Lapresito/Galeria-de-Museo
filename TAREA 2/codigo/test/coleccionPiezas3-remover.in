crearColeccionPiezas
crearFecha 1/1/2001
crearPieza 125 SacraMangostaCaliente Luna Starlight
agregarColeccionPiezas
removerDeColeccionPiezas 125
imprimirColeccionPiezas
crearFecha 1/1/2001
crearPieza 12 SacraLataFunebre Luna Starlight
agregarColeccionPiezas
crearFecha 1/1/2001
crearPieza 24 MalditaEspadaFurtiva Luna Starlight
agregarColeccionPiezas
crearFecha 1/1/2001
crearPieza 15 LaxaMangostaSilenciosa Luna Starlight
agregarColeccionPiezas
imprimirColeccionPiezas
removerDeColeccionPiezas 15
imprimirColeccionPiezas
removerDeColeccionPiezas 24
imprimirColeccionPiezas
removerDeColeccionPiezas 12
imprimirColeccionPiezas
liberarColeccionPiezas
Fin
