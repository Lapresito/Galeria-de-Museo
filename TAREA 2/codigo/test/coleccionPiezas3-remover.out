Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Coleccion de piezas creada de forma exitosa.
2> Fecha creada en forma exitosa.
3> Pieza creada en forma exitosa.
4> Pieza agregada de forma exitosa.
5> Pieza con id 125 removida con exito.
6> 7> Fecha creada en forma exitosa.
8> Pieza creada en forma exitosa.
9> Pieza agregada de forma exitosa.
10> Fecha creada en forma exitosa.
11> Pieza creada en forma exitosa.
12> Pieza agregada de forma exitosa.
13> Fecha creada en forma exitosa.
14> Pieza creada en forma exitosa.
15> Pieza agregada de forma exitosa.
16> Pieza: 12
SacraLataFunebre-Luna Starlight
Creada en: 1/1/2001
Pieza: 15
LaxaMangostaSilenciosa-Luna Starlight
Creada en: 1/1/2001
Pieza: 24
MalditaEspadaFurtiva-Luna Starlight
Creada en: 1/1/2001
17> Pieza con id 15 removida con exito.
18> Pieza: 12
SacraLataFunebre-Luna Starlight
Creada en: 1/1/2001
Pieza: 24
MalditaEspadaFurtiva-Luna Starlight
Creada en: 1/1/2001
19> Pieza con id 24 removida con exito.
20> Pieza: 12
SacraLataFunebre-Luna Starlight
Creada en: 1/1/2001
21> Pieza con id 12 removida con exito.
22> 23> Coleccion liberada con exito.
24> Fin.
